src/sentry/workflow_engine/processors/action.py

```diff
@@ -83,8 +83,6 @@ def create_workflow_fire_histories(
             workflow_id=workflow_id,
             group=event_data.event.group,
             event_id=event_data.event.event_id,
-            has_passed_filters=True,
```

Comment:
are you planning to also drop these columns in an upcoming migration?

Reply:
yes sir